cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/CgmesImportPostProcessorTest.java

```diff
@@ -28,6 +29,7 @@
 /**
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  */
+// FIXME(Luma) rename to CgmesImportProcessorsTest (we will test both pre- and post- processors)
```

Comment:
You can rename it now :)

Reply:
Yes, I forgot to do it. I have renamed the test class as `CgmesImportPreAndPostProcessorsTest`